modules/sdk-core/src/bitgo/tss/common.ts

```diff
@@ -95,7 +95,7 @@ export async function commonVerifyWalletSignature(params: {
     throw new Error('Invalid HSM GPG signature');
   }
 
-  const verificationResult = await walletSignature.verifyPrimaryUser([bitgoPub]);
+  const verificationResult = await walletSignature.verifyPrimaryUser([bitgoPub], null as unknown as undefined);
```

Comment:
From our convo on slack: 

There are many places we call `verifyPrimaryUser`. To make it explicit I think we should create some wrapper methods in our SDK like `verifyPrimaryUserWIthDateCheck` and `verifyPrimaryUserWithoutDateCheck`. 

The behaviour of the existing usages would remain the same via `verifyPrimaryUserWIthDateCheck` but it will be more clear to readers of the code what is happening in case this same bug re-appears in other places.